src/content/testimonials.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slides": [
+        {
+            "name": "Name 1",
+            "personDescription": "Position 1",
+            "content": "\"words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words words\"",
```

Comment:
It would be nice to place the content in quotes dynamically (in JS) as opposed to making every quote have a \" at the beginning and end. This way would be less prone to error in the future IMO